plugins/rating/test/rating.spec.js

```diff
@@ -198,6 +201,79 @@ test('gets aggregated rating stats with multiple labels', async (t) => {
   checkStatObject(results[0]['main:pricing'])
 })
 
+// use serial because no changes must be made during the check
+test.serial('check history filters', async (t) => {
+  const authorizationHeaders = await getAccessTokenHeaders({
+    t,
+    permissions: [
+      'rating:stats:all',
+      'rating:list:all',
+    ]
+  })
+
+  const groupBys = [
+    'authorId', // document first-level property
+    'assetId', // document data sub-property
+  ]
+
+  for (const groupBy of groupBys) {
+    const { body: { results } } = await request(t.context.serverUrl)
+      .get('/ratings?nbResultsPerPage=100')
+      .set(authorizationHeaders)
+      .expect(200)
+
+    const resultsByGroupBy = _.groupBy(results, groupBy)
+
+    const customExactValueCheck = _.curry((prop, obj, value) => {
+      let filteredResults = resultsByGroupBy[obj[groupBy]] || []
+      filteredResults = filteredResults.filter(r => r[prop] === value)
+      return filteredResults.length === obj.count
+    })
+
+    const customArrayValuesCheck = _.curry((prop, obj, values) => {
+      let filteredResults = resultsByGroupBy[obj[groupBy]] || []
+      filteredResults = filteredResults.filter(r => values.includes(r[prop]))
+      return filteredResults.length === obj.count
+    })
+
+    await checkFilters({
+      t,
+      endpointUrl: `/ratings/stats?groupBy=${groupBy}`,
+      fetchEndpointUrl: '/ratings',
+      authorizationHeaders,
+      checkPaginationObject: checkCursorPaginatedListObject,
+
+      filters: [
+        {
+          prop: 'authorId',
+          isArrayFilter: true,
+          customCheck: customArrayValuesCheck('authorId'),
+        },
+        {
+          prop: 'targetId',
+          isArrayFilter: true,
+          customCheck: customArrayValuesCheck('targetId'),
+        },
+        {
+          prop: 'assetId',
+          // isArrayFilter: true,
```

Comment:
Why do you keep these comments with `isArrayFilter` set to true (it’s false by default).

Reply:
Thanks for the catch. The comments has to be removed after removing rating array filter.